pilot/pkg/model/jwks_resolver.go

```diff
@@ -172,7 +172,7 @@ func newJwksResolverWithCABundlePaths(evictionDuration, refreshInterval time.Dur
 		},
 	}
 
-	caCertPool := x509.NewCertPool()
+	caCertPool := x509.SystemCertPool()
```

Comment:
the `SystemCertPool()` also returns an error variable: https://pkg.go.dev/crypto/x509#SystemCertPool. you can just log the error if any.